test/langtools/tools/javac/patterns/PrimitivePatternsSwitchRequirePreviewBoolean.java

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Having tests like this is not wrong, but instead of introducing 8 new (small) files, I would rather use the `ToolBox` and `TestRunner`, or a combo test. A `TestRunner` test looks like this:
https://github.com/openjdk/jdk/blob/76f792b55263faf883e54cb879d8609f87164e51/test/langtools/tools/javac/recovery/AttrRecovery.java